### PR TITLE
users: add Locha story

### PR DESCRIPTION
**Pull request by waehlisch**
add the user story of Locha Mesh and update layout elements in the context of user stories.

the logo wall is automatically generated by showing all images in `assets/img/use-cases/logos`

### CODE REVIEW

File: _use_cases/use-locha.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
+---
+project: Chat and send Bitcoin without Internet
+quote: In Locha Mesh, we decided to use RIOT for a variety of reasons, including an active community surrounding it, the overall design of the Operating System that's coupled with a powerful and extensible network stack, and the support for a good variety of CPUs and boards.
+user: Jean Pierre Dudey
+user_position: Embedded Software Developer
+user_photo: user-photos/Jean-Pierre-Dudey.jpg
+company: Loca Mesh
```
Comment (miri64) [inline _use_cases/use-locha.md:7]:
```suggestion
company: Locha Mesh
```

Comment (waehlisch) [reply _use_cases/use-locha.md]: thx. fixed.

**Comment by waehlisch** [comment]
ping @jia200x @leandrolanzieri @miri64

**Review by leandrolanzieri — APPROVED** [review]
ACK

File: _includes/use-cases-logo-wall.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
+        <div class="row row-cols-xxl-6 row-cols-lg-4 row-cols-md-3 row-cols-sm-3 row-cols-3">
+          {% for logo in site.static_files %}
+            {% if logo.path contains 'use-cases/logos' %}
+            <div class="col">
+              <div class="card border-0 d-flex h-100 p-4">
+                <div class="card d-flex align-items-center justify-content-center h-100 border-0">
+                  <img class="card-img-top" src="{{ logo.path }}" alt="">
```
Comment (leandrolanzieri) [inline _includes/use-cases-logo-wall.html:7]: Should we consider in the future using collections instead? Then we could also easily extend this with a link to the company and a proper alternate text, for example.

**Pull request merged**